cmd/terramate/cli/backend_cfg_gen_test.go

```diff
@@ -347,7 +354,8 @@ func TestBackendConfigGeneration(t *testing.T) {
   backend "fromparent" {
     attr = "value"
   }
-}`,
+}
```

Comment:
I'm reverting those newline changes... they're leftovers from the previous implementation. sorry